phylogenetic/defaults/auspice_config.json

```diff
@@ -66,6 +66,10 @@
     "distance_measure": "div",
     "transmission_lines": false
   },
+  "metadata_columns": [
+    "strain",
+    "url"
```

Comment:
_non-blocking_

FWIW, the `accession` and `url` fields should be [automatically included in export](https://github.com/nextstrain/augur/blob/8920730ab1b79124f66e55f65759c0aae745043d/augur/export_v2.py#L864) and so `url` does not need to be specified as additional `metadata_columns`.

Reply:
Thanks for linking to the Augur code to explain `url` field handling!

Indeed, `url` still works when I remove it from metadata_columns, so I’ve updated the commit accordingly.